user_docs/en/changes.md

```diff
@@ -90,7 +92,6 @@ There have also been a number of fixes, including to mouse tracking in Firefox,
 * eSpeak NG has been updated to 1.52-dev commit `961454ff`. (#16775)
   * Added new languages Faroese and Xextan.
 * When using a multi-line braille display via the standard HID braille driver, all lines of cells will be used. (#16993, @alexmoon)
-* The stability of NVDA's Poedit support has been improved with the side effect that the minimum required version of Poedit is now version 3.5. (#16889, @LeonarddeR)
```

Comment:
please remove the change to 2024.4 change log

Reply:
No idea how that got deleted, whoops.